hugegraph-dist/src/assembly/static/conf/gremlin-server.yaml

```diff
@@ -3,7 +3,7 @@
 #port: 8182
 
 # timeout in ms of gremlin query
-scriptEvaluationTimeout: 30000
+evaluationTimeout: 30000
```

Comment:
don't add files like: hugegraph-dist/src/assembly/static/conf/hugegraph-community.license

Reply:
Hi, I don't understand here,  the config is changed on the newest thinkerpop version.

Reply:
There are extra file "community.license" before, because I can't leave a comment on the binary file, so I put it here.